extractor/iscn.go

```diff
@@ -47,30 +43,31 @@ func insertISCN(payload db.EventPayload) error {
 	if err := json.Unmarshal(data.Record, &record); err != nil {
 		return fmt.Errorf("Failed to unmarshal iscn: %w", err)
 	}
-	stakeholdersEntities := []db.Entity{}
-	for _, stakeholder := range record.Stakeholders {
-		stakeholdersEntities = append(stakeholdersEntities, stakeholder.Entity)
-	}
-	stakeholdersJSON, err := json.Marshal(stakeholdersEntities)
-	if err != nil {
-		return fmt.Errorf("Failed to marshal sktaeholders: %w", err)
+	stakeholders := []db.Stakeholder{}
+	for _, stakeholderRawJSON := range record.Stakeholders {
+		parsedStakeholder := db.Stakeholder{}
+		err := json.Unmarshal(stakeholderRawJSON, &parsedStakeholder)
```

Comment:
What would happen if someone just decide to put invalid number, string, or big number/underfloat for rewards here?
Can we make rewards propotion `0` for invalid and missing cases?

Reply:
Better simply have the `data` JSONB field in the row, so we can query those using Postgres JSONB functions if needed, and indexes for `id` and `name`.

```SQL
CREATE TABLE iscn_stakeholders (
  -- pid means primary key ID, which is the auto-increment ID assigned by database in `iscn` table
  iscn_pid BIGINT REFERENCES iscn (id),
  -- stakeholder's ID, `sid` so not to be ambiguous with `id` column in `iscn` table
  sid TEXT,
  -- stakeholder's name, `sname` so not to be ambiguous with `name` column in `iscn` table
  sname TEXT,
  -- raw stakeholder object
  data JSONB
);
```

Reply:
Sure, as long as it works well reasonable speed
in case we change our mind later we can figure out how to migrate again

Reply:
e.g.
```SQL
SELECT
  contribution_type,
  COUNT(contribution_type)
FROM (
  SELECT
    (data ->> 'contributionType') as contribution_type
  FROM iscn_stakeholders
) t
GROUP BY contribution_type
ORDER BY count DESC
;
```
This runs blazing fast (< 0.5s), I don't know why...
(Maybe since I have just run the migration, so most of the pages are still in memory?)

Reply:
we only have a few type of `contribution_type` i guess (unlike stakeholder madness)